src/components/IndividualCharacter.js

```diff
@@ -0,0 +1,49 @@
+import { useHistory, useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import "./IndividualCharacter.css";
+
+export default function IndividualCharacter() {
+  const history = useHistory();
+  const { id } = useParams();
+  const [character, setCharacter] = useState({});
+
+  useEffect(() => {
+    const url = `https://rickandmortyapi.com/api/character/${id}`;
+
+    fetch(url)
+      .then((res) => res.json())
+      .then((apiData) => {
+        setCharacter(apiData);
+      });
+  }, [id]);
+
+  function capitalize(status) {
+    if (status) {
+      return status[0].toUpperCase() + status.slice(1);
+    }
+  }
```

Comment:
uuuh yeah! Nice work
This could also just be a general function in the `utils` folder of this app, I'd call it `capitalizeFirstLetter`